GCRCatalogs/SCHEMA.md

```diff
@@ -144,22 +153,12 @@ Quantity Label | Unit | Definition | GCRbase | DPDD
 `blendedness` | - | measure of how flux is affected by neighbors: (1 - flux.child/flux.parent) (see 4.9.11 of [1705.06766](https://arxiv.org/abs/1705.06766)) |   |   |
 `extendedness` | - | 0:star, 1:extended.  DM Stack `base_ClassificationExtendedness_value` |   |   |
 
-## Schema for DC2 Source Catalogs
+### Schema for DC2 Source and Forced Source Catalogs
 
-The schema for DC2 Schema Catalogs follow the following rules:
+Source Catalog contains information about high signal-to-noise detections on single frame images,
 
-- For quantities that are defined in [LSST DPDD](https://lse-163.lsst.io/dpdd.pdf), we follow DPDD's naming scheme.
-- For quantities that are defined in the above "Schema for Extragalatic Catalogs", we follow Extragalatic Catalogs' naming scheme ('GCRbase' below).
-- For quantities that are defined in both, we provide aliases so both naming schemes would work.
-- For quantities that are defined in neither and are newly defined for the coadd catalogs, we generally follow Extragalatic Catalogs' naming style.
-
-For quantities that are not yet documented in the table above, we document them below:
-- In the table below we list the name of the quantity, its units and definition and whether the name is defined in the GCRbase or DPDD.
-- Items marked with 'xx' are not exactly defined in the DPDD, but their name is taken from a related column in a different table.  E.g.
-   * there is no `x`, `y` in the DPDD Object table, but these are called `x`, '`y` in the DPDD Source table.  We don't have `xyCov` so we separately list `xErr` and `yErr`.
-   * `radec` is a pair in the DPDD, but we separate out into `ra`, `dec` here.
-   * The DPDD says `psCov`, but we only have the diagonal terms, so we call it `psErr`.
-- Quantities named with `x` or `y` are in the local coordinate system of the tract+patch.
+Forced Source Catalog contains photometry measurements about low signal-to-noise detections done on individual
```

Comment:
The Forced Source Catalog contains information on all Objects.

I think mentioning "low signal-to-noise detections" is confusing here, both "low signal-to-noise" and "detections".  

It's potentially of significant interest for low SNR things that may or may not have yielded detections on the individual image.
In the high SNR regime, the Forced Source Catalog entries should agree with the Source Catalog entries.

"Forced Source Catalog contains photometry based on fixed-position measurements from objects in the Object Table."

Reply:
Thanks, @wmwv --- do we want to mention that it is done on individual exposures? 

"Forced Source Catalog contains photometry from individual exposures based on fixed-position measurements from objects in the Object Table."

Reply:
How about:"Forced Source Catalog contains fixed-position photometry measurements performed on individual exposures based on the positions from objects in the Object Table."

Reply:
Updated. Ready for re-review. Thanks!